### PR TITLE
Added questions to General section

### PR DESCRIPTION
**Pull request by anika-kamath**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -21,6 +21,17 @@ const generalCSQuiz = [
...
   },
+  {
+    Question:
+      "Which of these sorting algorithms follows an average case time complexity of order O(nlogn)?",
+    Answer: "Merge Sort",
+    Distractor1: "Selection Sort",
+    Distractor2: "Bubble Sort",
+    Distractor3: "Insertion Sort",
+    Explanation:
+      "The average case time complexity of Merge Sort Algorithm is O(nlogn) whereas for the other 3 options it is O(n^2).",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:32]: I don't think Merge Sort Algorithm needs to be capitalized. 

```suggestion
      "The average case time complexity for the merge sort algorithm is O(n log n).",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by anika-kamath** [comment]
@jdwilkin4 I have made the suggested changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**